devito/ir/iet/visitors.py

```diff
@@ -261,11 +259,10 @@ def visit_Iteration(self, o):
             loop_inc = '%s += %s' % (o.index, o.limits[2])
 
         # Append unbounded indices, if any
-        if o.uindices:
-            uinit = ['%s = %s' % (i.index, ccode(i.start)) for i in o.uindices]
-            loop_init = c.Line(', '.join([loop_init] + uinit))
-            ustep = ['%s = %s' % (i.index, ccode(i.step)) for i in o.uindices]
-            loop_inc = c.Line(', '.join([loop_inc] + ustep))
+        uindex_init = [c.Initializer(c.Value('int', i.index), ccode(i.step))
+            for i in o.uindices]
```

Comment:
perhaps flake8 will complain? probably ``for i in o.uindices`` should start under `c` in `c.Initializer...`

Reply:
Yeah, it already did :-)  I was just fixing that one up....